src/python/pants_release/generate_github_workflows.py

```diff
@@ -19,6 +19,23 @@
 
 from pants.util.strutil import softwrap
 
+ACTION = {
+    "action-send-mail": "dawidd6/action-send-mail@v3.8.0",
+    "cache": "actions/cache@v4",
+    "checkout": "actions/checkout@v4",
+    "download-artifact": "actions/download-artifact@v4",
+    "expose-pythons": "pantsbuild/actions/expose-pythons@627a8ce25d972afa03da1641be9261bbbe0e3ffe",
+    "github-action-required-labels": "mheap/github-action-required-labels@v4.0.0",
+    "rust-cache": "benjyw/rust-cache@461b9f8eee66b575bce78977bf649b8b7a8d53f1",
+    "setup-go": "actions/setup-go@v5",
+    "setup-java": "actions/setup-java@v4",
+    "setup-node": "actions/setup-node@v4",
+    "setup-protoc": "arduino/setup-protoc@9b1ee5b22b0a3f1feb8c2ff99b32c89b3c3191e9",
+    "setup-python": "actions/setup-python@v5",
+    "slack-github-action": "slackapi/slack-github-action@v1.24.0",
+    "upload-artifact": "actions/upload-artifact@v4",
```

Comment:
At $WORK, we were tripped up by one of the breaking changes in this upgrade: https://github.com/actions/upload-artifact?tab=readme-ov-file#breaking-changes

> Uploading to the same named Artifact multiple times.
>
> Due to how Artifacts are created in this new version, it is no longer possible to upload to the same named Artifact multiple times. You must either split the uploads into multiple Artifacts with different names, or only upload once. Otherwise you will encounter an error.

Does Pants encounter this?

Maybe that is this why there's the `overwrite: true` added in `upload_log_artifacts`?

Reply:
yea, that was my attempt at handling that, suspecting that we want the "latest" logs always. Other than that 🤷🏽  so figured I run some workflows manually using this version, once we're passed the other CI issues (the failing build wheels jobs).